src/SeqViewerContainer.tsx

```diff
@@ -46,11 +46,13 @@ interface SeqViewerContainerProps {
   rotateOnScroll: boolean;
   search: NameRange[];
   selectAllEvent: (event: React.KeyboardEvent<HTMLElement>) => boolean;
-  selection?: {
-    clockwise?: boolean;
-    end: number;
-    start: number;
-  };
+  selection?:
```

Comment:
That was my understanding as well, but for some reason the function`componentDidUpdate = (prevProps: SeqViewerContainerProps)` gets called when a user clicks on an element, and in `this.props.selection` you see the selection with type and the other fields.

In any case, I removed this since the type guard takes care of checking if it's a Selection or not.